superset/initialization/__init__.py

```diff
@@ -53,7 +53,7 @@
     stats_logger_manager,
     talisman,
 )
-from superset.security import SupersetSecurityManager
+from superset.security.manager import SupersetSecurityManager
```

Comment:
After the other changes, this line complained that SupersetSecurityManager did not exist. This is where other files import it from.